packages/ilib-po/tsconfig.json

```diff
@@ -1,21 +1,12 @@
 {
+    "extends": "@tsconfig/node12",
     "compilerOptions": {
-        "allowJs": false, // https://github.com/microsoft/TypeScript/issues/35470,
-        "baseUrl": "./",
-        "moduleResolution": "node",
-        "checkJs": false,
-        "esModuleInterop": true,
-        "jsx": "react",
-        "module": "ES2020",
+        "allowJs": true,
+        "noEmit": true,
         "noFallthroughCasesInSwitch": true,
         "noImplicitOverride": true,
-        "paths": {
-            "*": ["node_modules/*", "src/*"]
-        },
-        "resolveJsonModule": true,
-        "target": "ES5",
```

Comment:
Removing this gets rid of the transpilation for older versions of node, which we had agreed upon. Transpilation should be done on a modern version of node instead of the older ones...

Reply:
Actually no, I just removed this explicit configuration in favour of preconfigured `@tsconfig/node12` https://github.com/tsconfig/bases#readme

Reply:
Please also note that I've added devdep `"@types/node": "12"` which provides type definitions available in Node 12 specifically (which helped me catch unsupported `node:` prefix in https://github.com/iLib-js/ilib-mono/pull/23/commits/6c352ca781c90f13ae222a57f12c21be1babde6a#diff-50c15f283c9d454424a9c075add5bc25c9aaa034a20e4a940881645c69196ab7R20 since apparently it's been added in v16 https://2ality.com/2021/12/node-protocol-imports.html)